README.md

```diff
@@ -78,57 +78,71 @@ print(await sandbox.execute("float(x[0])", session_id="123"))
 You can use `PyodideSandbox` as a LangChain tool:
 
 ```python
-from langchain_sandbox import PyodideSandboxTool
+from langchain_sandbox import PyodideStatelessSandboxTool
 
-tool = PyodideSandboxTool()
+tool = PyodideStatelessSandboxTool()
 result = await tool.ainvoke("print('Hello, world!')")
 ```
 
-If you want to persist state between code executions (to persist variables, imports,
-and definitions, etc.), you need to invoke the tool with `thread_id` in the config:
+### Using with an agent
+
+You can use sandbox tools inside a LangGraph agent:
 
 ```python
-code = """\
-import numpy as np
-x = np.array([1, 2, 3])
-print(x)
-"""
-result = await tool.ainvoke(
-    code,
-    config={"configurable": {"thread_id": "123"}},
-)
+from langgraph.prebuilt import create_react_agent
+from langchain_sandbox import PyodideStatelessSandboxTool, PyodideSandbox
 
-second_result = await tool.ainvoke(
-    "print(float(x[0]))",  # tool is aware of the previous result
-    config={"configurable": {"thread_id": "123"}},
+sandbox = PyodideSandbox(allow_net=True)
+tool = PyodideStatelessSandboxTool(sandbox=sandbox)
```

Comment:
Do we need to create a sandbox instance vs. passing in configuration to initialize a sandbox?

e.g.,

could we just expose `allow_net` and initialize a sandbox appropriately?

Reply:
This would also allow us to instantiate both sync and async sandboxes under the hood